framework/console/Controller.php

```diff
@@ -87,6 +87,17 @@ public function runAction($id, $params = [])
         if (!empty($params)) {
             // populate options here so that they are available in beforeAction().
             $options = $this->options($id === '' ? $this->defaultAction : $id);
+            if (isset($params['alias'])) {
+                $optionAliases = $this->optionAliases();
+                foreach ($params['alias'] as $name => $value) {
+                    if (key_exists($name, $optionAliases)) {
```

Comment:
Should be `array_key_exists` because it's preferred to use real names, not aliases.